server/community.go

```diff
@@ -64,3 +66,62 @@ func (s *Server) postPRWelcomeMessage(ctx context.Context, pr *model.PullRequest
 
 	return nil
 }
+
+func (s *Server) assignGreeter(ctx context.Context, pr *model.PullRequest, repo *Repository) error {
+	// Only assign an greeter for non-member PRs
+	if s.IsOrgMember(pr.Username) {
+		return nil
+	}
+
+	// Is it configured to have a greeting team to handle the PR?
+	if repo.GreetingTeam == "" {
+		return nil
+	}
+
+	// get available members
+	greeterList, _, err := s.GithubClient.Teams.ListTeamMembersBySlug(ctx, s.Config.Org, repo.GreetingTeam, nil)
+	if err != nil {
+		mlog.Error("Error retrieving greeting team members", mlog.Err(err), mlog.String("GreetingTeam", repo.GreetingTeam))
+		return err
+	}
+
+	// Assign one of them as a reviewer
+	size := len(greeterList)
+	if size == 0 {
+		mlog.Warn("There were no members on the greeting team", mlog.String("GreetingTeam", repo.GreetingTeam))
+		return nil
+	}
+
+	greeter := greeterList[rand.Intn(size)]
+	greetingRequest := github.ReviewersRequest{
+		Reviewers: []string{*greeter.Login},
+	}
+
+	_, _, err = s.GithubClient.PullRequests.RequestReviewers(ctx, pr.RepoOwner, pr.RepoName, pr.Number, greetingRequest)
+	if err != nil {
+		mlog.Warn("Couldn't assing a member of the greeting team", mlog.Err(err), mlog.String("GreetingTeam", repo.GreetingTeam))
+		return nil
+	}
+
+	return nil
+}
+
+func (s *Server) assignGreetingLabels(ctx context.Context, pr *model.PullRequest, repo *Repository) error {
+	// Only assign labels for non-member PRs
+	if s.IsOrgMember(pr.Username) {
+		return nil
+	}
+
+	// Is repo configured to setup labels for non-member PRs?
+	if len(repo.GreetingLabels) == 0 {
+		return nil
+	}
+
+	// Assign labels
+	_, _, err := s.GithubClient.Issues.AddLabelsToIssue(ctx, pr.RepoOwner, pr.RepoName, pr.Number, repo.GreetingLabels)
```

Comment:
Does this handle the case where the labels are already attached to the issue? For example, I use the `gh` CLI tool to automatically assign labels and reviewers when I create a PR. Some contributors might also do the same.